server_src/runners/history.js

```diff
@@ -236,30 +232,27 @@ class HistoryCollection {
               "Successfully grabbed timelapse list... Checking for:",
               fileName
             );
-            this.unrenderedFileName = null;
-            console.log("Check", fileName);
+            let unrenderedFileName = null;
             if (timelapseResponse.unrendered.length === 0) {
               let cleanName = fileName;
               if (fileName.includes(".gcode")) {
                 cleanName = fileName.replace(".gcode", "");
               }
               let lastTimelapse = null;
-              console.log("Checking for", this.unrenderedFileName);
-              if (this.unrenderedFileName === null) {
-                console.log(
-                  `Unrendered is ${this.unrenderedFileName}... hmmm... fallback to name match on last index...`
+              if (unrenderedFileName === null) {
```

Comment:
Here and below I just cleaned up my incorrect usage of "this"

Reply:
This, that, them, they, where does it end? ;)

Reply:
unrenderedFileName will always be null?